one-ui/src/main/java/com/marklogic/hub/oneui/services/DataHubProjectUtils.java

```diff
@@ -35,17 +35,34 @@
 import org.springframework.web.multipart.MultipartFile;
 
 public class DataHubProjectUtils {
```

Comment:
The reason to move the utility class to services package is that "no such method found" thrown if it is put into utils package during junit test case was run. I did not figure out the exact reason...